framework/resources/Functional/metadata_caching/generated_caches/validate_cache1.e_tsv

```diff
@@ -1 +1 @@
+v3	\{"varchar_col":\{"name":\["varchar_col"\],"primitiveType":"BINARY","originalType":"UTF8","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"timestamp_col":\{"name":\["timestamp_col"\],"primitiveType":"INT64","originalType":"TIMESTAMP_MILLIS","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"time_col":\{"name":\["time_col"\],"primitiveType":"INT32","originalType":"TIME_MILLIS","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"bigint_col":\{"name":\["bigint_col"\],"primitiveType":"INT64","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"date_col":\{"name":\["date_col"\],"primitiveType":"INT32","originalType":"DATE","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"int_col":\{"name":\["int_col"\],"primitiveType":"INT32","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"interval_col":\{"name":\["interval_col"\],"primitiveType":"BINARY","originalType":"UTF8","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"bool_col":\{"name":\["bool_col"\],"primitiveType":"BOOLEAN","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"double_col":\{"name":\["double_col"\],"primitiveType":"DOUBLE","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\},"float_col":\{"name":\["float_col"\],"primitiveType":"FLOAT","precision":"0","scale":"0","repetitionLevel":"0","definitionLevel":"0"\}\}	\[\{"path":"/drill/testdata/metadata_caching/fewtypes/fewtypes.parquet","length":"2457","rowGroups":\[\{"start":"4","length":"1661","rowCount":"21","hostAffinity".*"columns":\[\{"name":\["int_col"\],"minValue":"1","maxValue":"21","nulls":"0"\},\{"name":\["bigint_col"\],"minValue":"302","maxValue":"98723980547","nulls":"0"\},\{"name":\["date_col"\],"minValue":"-4650","maxValue":"25667","nulls":"0"\},\{"name":\["time_col"\],"minValue":"0","maxValue":"86399990","nulls":"0"\},\{"name":\["timestamp_col"\],"minValue":"-2299559279000","maxValue":"4011874321000","nulls":"0"\},\{"name":\["interval_col"\]\},\{"name":\["varchar_col"\]\},\{"name":\["float_col"\],"minValue":"23.5","maxValue":"2.34555238E9","nulls":"0"\},\{"name":\["double_col"\],"minValue":"654.4","maxValue":"2.22223454324544E12","nulls":"0"\},\{"name":\["bool_col"\],"minValue":"false","maxValue":"true","nulls":"0"\}\]\}\]\}\]	1.10.0-SNAPSHOT
```

Comment:
There are two reasons why MD-1181 is important. One is the effort needed every 3 months or so. But if you are willing to do that, then yeah, its okay to delay. 

But during a release of say 1.10.0, for a few weeks, we will have both 1.10.0-SNAPSHOT and 1.10.0 exist - one in the master and one in the release branch. This means during every release we will have these as false negatives. We will not be able to update baseline to 1.10.0 - and will need to wait until master moves to 1.11.0-SNAPSHOT before we can update baselines.